src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -96,16 +96,183 @@ class DWTConfig(EventDispatcher):
     active_config = ObjectProperty(config_classes.Configuration.default())  # type: config_classes.Configuration
     active_cutter = ObjectProperty(config_classes.Cutter.default())  # type: config_classes.Cutter
 
-    def __init__(self, screen_drywall_cutter=None, *args, **kwargs):
+    shape_options_dict = {
+        'circle': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/circle_shape_button.png',
+        }, 'square': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/square_shape_button.png',
+        }, 'line': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/line_shape_button.png',
+        }, 'geberit': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/geberit_shape_button.png',
+        }, 'rectangle': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/rectangle_shape_button.png',
+        }
+    }
+
+    toolpath_offset_options_dict = {
+        'inside': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/toolpath_offset_inside_button.png',
+        }, 'outside': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/toolpath_offset_outside_button.png',
+        }, 'on': {
+            'image_path': './asmcnc/apps/drywall_cutter_app/img/toolpath_offset_on_button.png',
+        }
+    }
+
+    shape_rules = {
+        'line': {
+            'toolpath_offset': ['on'],
+            'rotate_enabled': True,
+            'tool_whitelist': None
+        },
+        'rectangle': {
+            'toolpath_offset': ['inside', 'on', 'outside'],
+            'rotate_enabled': True,
+            'tool_whitelist': None
+        },
+        'square': {
+            'toolpath_offset': ['inside', 'on', 'outside'],
+            'rotate_enabled': False,
+            'tool_whitelist': None
+        },
+        'circle': {
+            'toolpath_offset': ['inside', 'on', 'outside'],
+            'rotate_enabled': False,
+            'tool_whitelist': None
+        },
+        'geberit': {
+            'toolpath_offset': ['on'],
+            'rotate_enabled': True,
+            'tool_whitelist': ['tool_6mm.json', 'tool_8mm.json']
+        }
+        # Add new shape rules here
+    }
+
+    def __init__(self, screen_drywall_cutter, *args, **kwargs):
         super(DWTConfig, self).__init__(*args, **kwargs)
-        self.screen_drywall_cutter = screen_drywall_cutter
+        self.drywall_screen = screen_drywall_cutter
+
+        self.tool_options = self.get_available_cutter_names()
 
         most_recent_config_path = self.get_most_recent_config()
         if most_recent_config_path:
             self.load_config(most_recent_config_path)
         else:
             self.load_temp_config()
 
+        self.setup_dropdowns()
+
+    def setup_dropdowns(self):
+        self.drywall_screen.tool_selection.set_image(
```

Comment:
Rename tool_selection and others to reflect their class? I think it would make it clearer.